### DISCUSSION TITLE
Move some examples to a new section in the Docs

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pastas/pastas#248] [Ideas]
There is an increasing number of examples available on the Documentation website. However, quite a few of them are actually not examples, but proof of concepts or summaries of what is available. I suggest to move these to a new folder named "concepts", that will then show up under "User Guide" on the Docs. We may also rename "User Guide" to "Concepts".

Concept notebooks:
- 2_TimeSeries_objects.ipynb
- 3_response_functions.ipynb
- 8_pastas_synthetic.ipynb
- 15_armamodel.ipynb
- 16_noisemodel.ipynb
- 17_Autocorrelation.ipynb

I think this would make it much easier for newcomers to find examples on how to USE Pastas, instead of very technical notebooks describing some underlying principles. On the other hand, those interested in the latter will find them easier as well. Thoughts?

**Comment by dbrakenhoff** [comment]
Sounds like a good idea to me!

**Comment by mbakker7** [comment]
+1

**Comment by raoulcollenteur** [comment]
Cool, working on this now.
